cpp/Dockerfile.alpine

```diff
@@ -32,6 +32,7 @@ RUN apk add --no-cache -q \
         make \
         musl-dev \
         ninja \
+        openssl-dev \
```

Comment:
This was needed for building Flight/gRPC. I have not yet activated it in the Alpine build as the tests don't pass.